server/util/redact/redact.go

```diff
@@ -213,6 +224,16 @@ func stripRepoURLCredentialsFromBuildMetadata(metadata *bespb.BuildMetadata) {
 			metadata.Metadata[repoURLKey] = gitutil.StripRepoURLCredentials(val)
 		}
 	}
+	if m, ok := metadata.Metadata[explicitCommandLineName]; ok {
+		var commandLine []string
+		json.Unmarshal([]byte(m), &commandLine)
+		redactCmdLine(commandLine)
+		commandLineJSON, err := json.Marshal(commandLine)
+		if err != nil {
+			metadata.Metadata[explicitCommandLineName] = ""
+		}
+		metadata.Metadata[explicitCommandLineName] = string(commandLineJSON)
```

Comment:
I think this code is equivalent since the line after the err check runs unconditionally:

```suggestion
		commandLineJSON, _ := json.Marshal(commandLine)
		metadata.Metadata[explicitCommandLineName] = string(commandLineJSON)
```